src/test/librbd/test_DeepCopy.cc

```diff
@@ -512,6 +512,7 @@ TEST_F(TestDeepCopy, Stress)
 
 TEST_F(TestDeepCopy, NoSnaps_LargerDstObjSize)
 {
+  SKIP_IF_CRIMSON();
```

Comment:
Just noting that this would skip these tests unconditionally, despite

> Skip v1 LargerDstObjSize for Crimson until resolved.

in the PR description and

> librbd/test_DeepCopy: skip v2 LargerDstObjSize for crimson

in the commit title.  Is the issue with v1, v2 or both?

Reply:
> Just noting that this would skip these tests unconditionally

These tests causes the client to abort so they are disabled for now in order to test the suite properly.
```
[ RUN      ] TestDeepCopy.NoSnaps_LargerDstObjSize
free(): invalid pointer
[1]    2891009 abort 
```
The issue is with both v1 and v2. Will update the description/commit. Thank you!